tests/rados/test_bluestore_comp_enhancements_class.py

```diff
@@ -911,9 +913,20 @@ def create_pool_wrapper(self, pool_name: str):
             AssertionError: If pool creation fails.
             Exception: If the pool's pg_num does not reach 1 within 600 seconds.
         """
-        assert self.rados_obj.create_pool(
-            pool_name=pool_name, pg_num=1, disable_pg_autoscale=True, app_name="rbd"
-        )
+        if self.pool_type == "replicated":
+            assert self.rados_obj.create_pool(
+                pool_name=pool_name, pg_num=1, disable_pg_autoscale=True, app_name="rbd"
+            )
+        else:
+            config = {
+                "create": True,
+                "profile_name": "ec42_rados" + generate_unique_id(3),
+                "pool_name": pool_name,
+                "k": 4,
+                "m": 2,
+                "crush-failure-domain": "host",
```

Comment:
Update the failure domain to OSD. we need to have atleast k+M OSD hosts for this